servatrice/src/servatrice_database_interface.cpp

```diff
@@ -124,6 +124,8 @@ bool Servatrice_DatabaseInterface::execSqlQuery(QSqlQuery *query)
 bool Servatrice_DatabaseInterface::usernameIsValid(const QString &user, QString & error)
 {
     int minNameLength = settingsCache->value("users/minnamelength", 6).toInt();
+    if(minNameLength < 1)
```

Comment:
This is a sanity check to ensure the min length of a valid username is at least 1; it just defends servatrice from having a "0" written in its config file.